volumes/miovision/sql/function/function-clear-api_log.sql

```diff
@@ -11,12 +11,7 @@ COST 100
 AS $BODY$
 
 DECLARE
```

Comment:
should I comment out the raise notice here as well?

Reply:
It's only causing problems here, because of the `if conn.notices != []:` https://github.com/CityofToronto/bdit_data-sources/blob/a5355e16a5a4118ba4b7a66074552027bf4bc2b0/volumes/miovision/api/intersection_tmc.py#L439-L443